install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -870,6 +870,9 @@ data:
 {{- if (not (kindIs "invalid" .Values.cni.externalRouting)) }}
   cni-external-routing: {{ .Values.cni.externalRouting | quote }}
 {{- end}}
+{{- if (not (kindIs "invalid" .Values.cni.proxyRouting)) }}
```

Comment:
This seems to be missing from `values.yaml` -- can you add it, with documentation?